prep-css-box-model/style.css

```diff
@@ -0,0 +1,12 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+  border-width: 3px;
+  border-color: darkgray;
```

Comment:
We also want a double border style 😄

Reply:
I added and pushed through the new code but now I am having the same issue as the last time I showed you on Slack